crates/pop-parachains/templates/pallet/advanced_mode/src/tests.rs.templ

```diff
@@ -0,0 +1,7 @@
+use crate::{mock::*, *};
+use frame::deps::frame_support::{assert_noop, assert_ok};
```

Comment:
the `testing::prelude` should cover this.

Reply:
Committed!